docs/application/web/guides/device/feedback.md

```diff
@@ -16,6 +16,28 @@ The main features of the Feedback API include:
 
 Each feedback pattern can have separate media files of sound and vibration type. For example, when receiving a new email message, a predefined sound and vibration feedback pattern can be played.
 
+## Prerequisites
+
+1. Since Tizen 5.0 it is possible to make your application visible in the Tizen Store only for devices that support feedback vibration feature. For that purpose the application must specify the following feature in the `config.xml` file:
+
+   ```
+    <widget>
+       <feature name="http://tizen.org/feature/feedback.vibration"/>
+    </widget>
+   ```
+
+    Additionally, you can also verify if the feedback vibration feature is supported using the `tizen.systeminfo.getCapability()` method and accordingly enable or disable the code that needs the API:
```

Comment:
add a comma after method.

 Additionally, you can also verify if the feedback vibration feature is supported using the `tizen.systeminfo.getCapability()` method, and accordingly enable or disable the code that needs the API:

Reply:
Done